go.sum

```diff
@@ -28,8 +28,10 @@ github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5 h1:0CwZNZbxp69SHPd
 github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5/go.mod h1:wHh0iHkYZB8zMSxRWpUBQtwG5a7fFgvEO+odwuTv2gs=
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
 github.com/cespare/xxhash/v2 v2.1.1/go.mod h1:VGX0DQ3Q6kWi7AoAeZDth3/j3BFtOZR5XLFGgcrjCOs=
-github.com/ciscoecosystem/aci-go-client/v2 v2.0.2 h1:B/WwSMZ0CE/QgKvnIrKXPyj1g9f4OanY8aggULbwy2E=
-github.com/ciscoecosystem/aci-go-client/v2 v2.0.2/go.mod h1:ZeJuaKr9gTRn5B1CwA3sVSVqnRjVMEcBc8LUX222wf0=
+github.com/ciscoecosystem/aci-go-client/v2 v2.4.0 h1:oRzLLTzhYVMhMwn7Ef+vRSbe9lNem03S6qGL4VMRgX8=
```

Comment:
could be removed, new version is 2.6.2 already in master so probably best to rebase